backend-models/service-tests/accessedservice-tests.txt

```diff
@@ -0,0 +1,87 @@
+
+accessedservice.php
+
+-------------------
+
+********
+* pre-req
+********
+
+(checklogin() is TRUE && (hasAccess($userid, $cid, 'w') || isSuperUser($userid,) is TRUE))
+
+--------------------
+
+Test 1: try to update a created user by changing the username
```

Comment:
To be able to call this a test case we need actual values, not variable names. I would recommend to look at how this looks. https://github.com/HGustavs/LenaSYS/blob/26a2a6c97af16d1be7871ebf481d2a253056c7eb/backend-models/service-tests/duggaedservice-tests.txt#L23 

Here there are actual values prepared to be used for the test case